Aula02/style_blog.css

```
@@ -0,0 +1,93 @@
+.blog-footer{
+    background-color: black;
+    padding: 10px;
+    font-size: 30px;
+    display: flex;
+    justify-content: space-between;
+}
+
+.menu {
+    background-color: black;
+    font-size: 30px;
+    margin: 0;
+    padding: 0;
+    display: flex;
+    justify-content: space-between;
+}
+  
+.menu li {
+    display: inline-block;
+    margin-right: 20px;
+}
+  
+.menu li a {
+    display: block;
+    padding: 10px;
+    text-decoration: none;
+}
+  
+.dropdown:hover .dropdown-menu {
+    display: block;
+}
+  
+.dropdown-menu {
+    display: none;
+    position: absolute;
```

quando a gente usa um position absolute ele precisa de um relative pra facilitar a movimentação do objeto usando os atributos left, top, right e bottom. 

O relative precisa ser um dos pais do objeto, assim a gente consegue usar o left e o right como 0  para forçar ele a ter sempre a mesma largura do pai.

```suggestion
    position: absolute;
    left: 0;
    right: 0;
```

---

Ai as margins e os paddins tu ajeita ai com base no resultado que tu quer